modules/framework/galasa-parent/dev.galasa.framework/src/testFixtures/java/dev/galasa/framework/mocks/MockStream.java

```diff
@@ -67,19 +58,28 @@ public void setDescription(String description) {
         this.description = description;
     }
 
-    public void setMavenRepositoryUrl(String mavenRepositoryUrl) {
-        this.mavenRepositoryUrl = mavenRepositoryUrl;
+    public void setMavenRepositoryUrl(String mavenRepositoryUrl) throws MalformedURLException {
+        this.mavenRepositoryUrl = new URL(mavenRepositoryUrl);
     }
 
-    public void setTestCatalogUrl(String testCatalogUrl) {
-        this.testCatalogUrl = testCatalogUrl;
+    public void setTestCatalogUrl(String testCatalogUrl) throws MalformedURLException {
+        this.testCatalogUrl = new URL(testCatalogUrl);
     }
 
-    public void setObrLocation(String obrLocation) {
-        this.obrLocation = obrLocation;
+    public void setObrs(List<String> obrs) {
+        this.obrs = obrs;
     }
 
     public void setEnabled(boolean isEnabled) {
         this.isEnabled = isEnabled;
     }
+
+    @Override
+    public boolean isValid() {
+        boolean isValid = (
```

Comment:
implementation inside a test class ?

Reply:
Addressed in https://github.com/galasa-dev/galasa/pull/236/commits/a9ef197a47c1c7da97f4c1566f80ca24bce91184